test/functional/apps/discover/_indexpattern_without_timefield.ts

```diff
@@ -44,7 +42,6 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should display a timepicker after switching to an index pattern with timefield', async function() {
-      expect(await PageObjects.timePicker.timePickerExists()).to.be(false);
       await PageObjects.discover.selectIndexPattern('with-timefield');
       expect(await PageObjects.timePicker.timePickerExists()).to.be(true);
```

Comment:
Unrelated recommendation: I think avoiding boolean assertions makes failures a lot easier to debug. It's often unhelpful to get `expected false to be true`, but it's really easy to do this:

```ts
if (!await PageObjects.timePicker.timePickerExists()) {
  throw new Error('expected timepicker to exist')
}
```

Reply:
this is so true! thx for that! but shouldn't there be at least 1 expect in a test? or isn't it necessary?

Reply:
Nope, using `expect()` is not necessary to write meaningful assertions.

Reply:
thx, i've refactored it this way